packages/inventory/src/components/InventoryList/index.tsx

```diff
@@ -60,24 +60,26 @@ const InventoryList = (props: InventoryListProps) => {
     getInventoriesByExpiry(state, { servicePointIds: [servicePointId], expired: false })
   ) as Inventory[];
   const { broken, handleBrokenPage } = useHandleBrokenPage();
+  const [loading, setLoading] = useState(true);
```

Comment:
Am nitpicking here, setting loading to true initially doesn't feel right. The loading value is based on data being fetched remotely. This is one of the areas react query would do well.

Reply:
I was set on using least effort here since, this module code is part of the to be soon deprecated opensrp-web. That's why I did not choose to refactor this to react-query, since there are also the tests to think about.



> The loading value is based on data being fetched remotely

Now that we are using setLoading, the reason  I set it to true initally is because I know this view requires the data to be fetched for it to properly render, so setting true from the start, I think, prevents a possible flash of content in between the first 2 renders i.e. before the useEffect that would initially set loading to true is triggered.